pkg/intermediate/aggregate_test.go

```diff
@@ -82,7 +82,11 @@ func createMsgwithTemplateSet(isIPv6 bool) *entities.Message {
 	message.SetSequenceNum(1)
 	message.SetObsDomainID(5678)
 	message.SetExportTime(0)
-	message.SetExportAddress("127.0.0.1")
+	if isIPv6 {
+		message.SetExportAddress("2001:0:3238:BBBB:63::AAAA")
```

Comment:
is this localhost IPv6 address?

Reply:
It's not. Changed to localhost address `[::1]:0`. Thanks!